### PR TITLE
feat: adding required ccd fields (FPLA-768)

### PR DESCRIPTION
**Pull request by toireasam**
### Change description ###
Adding ccd required fields for populating collection

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: ccd-definition/AuthorisationCaseField/CareSupervision/superuser.json [hunk 1]
```diff
@@ -132,5 +132,19 @@
     "CaseFieldID": "showRemoveSDOWarningFlag",
     "UserRole": "caseworker-publiclaw-superuser",
     "CRUD": "CRU"
+  },
```
Comment (samuelzerozero) [inline ccd-definition/AuthorisationCaseField/CareSupervision/superuser.json:135]: that should move to systemupdate

**Review by samuelzerozero — APPROVED** [review]

**Review by samuelzerozero — APPROVED** [review]

File: ccd-definition/CaseField/CareSupervision/caseField.json [hunk 2]
```diff
@@ -1303,5 +1303,33 @@
...
+    "ID": "uploadApplicationDocumentsLabel",
+    "Label": "Select the supporting or annex documents you're sending with this application. <br> These can be part of the social work evidence template (SWET) or separate.",
+    "FieldType": "Label",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "applicationDocuments",
```
Comment (joez-f) [inline ccd-definition/CaseField/CareSupervision/caseField.json:1319]: maybe systemupdate user needs permissions? i will add during my commit 👍

File: ccd-definition/CaseField/CareSupervision/caseField.json [hunk 3]
```diff
@@ -1303,5 +1303,33 @@
     "FieldType": "Label",
     "SecurityClassification": "Public",
     "Searchable": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "uploadApplicationDocumentsLabel",
+    "Label": "Select the supporting or annex documents you're sending with this application. <br> These can be part of the social work evidence template (SWET) or separate.",
```
Comment (joez-f) [inline ccd-definition/CaseField/CareSupervision/caseField.json:1311]: technically we don't need to do this label in this PR right? I think PR should just be for the fields that need to be present as soon as we merge (i.e new fields that are being populated in old events). I guess the R permissions for other users are also not needed if you wanted PR to be smaller, but no harm in putting them

**Review by joez-f — APPROVED** [review]

File: ccd-definition/CaseField/CareSupervision/caseField.json (hunk 3 above)
Comment (toireasam) [reply ccd-definition/CaseField/CareSupervision/caseField.json]: removed

File: ccd-definition/CaseField/CareSupervision/caseField.json (hunk 2 above)
Comment (toireasam) [reply ccd-definition/CaseField/CareSupervision/caseField.json:1310]: leaving this to our bigger pr's as discussed

**Pull request merged**